src/cryptoadvance/specter/devices/bitcoin_core.py

```diff
@@ -256,3 +256,46 @@ def delete(
                         break
         except:
             pass  # We tried...
+
+
+class BitcoinCoreWatchOnly(BitcoinCore):
+    device_type = DeviceTypes.BITCOINCORE_WATCHONLY
+    name = "Bitcoin Core (watch only)"
+    icon = "bitcoincore_icon.svg"
+
+    hot_wallet = True
```

Comment:
Unnecessary since it inherits BitcoinCore which has it already

Reply:
Removed in latest commit